### PR TITLE
Feature/new tango ros release

### PR DESCRIPTION
**Pull request by PerrineAguiar**
This PR adds artifacts from the new release of TangoRosStreamer.
New packages:
* tango_nodelet_manager
* tango_ros_common
* tango_ros_messages (inside `org/ros/rosjava_messages`)

Note: 
* tango_ros_node is now obsolete (replaced by tango_nodelet_manager)
* the app apk was removed since it is an unsigned apk

### CODE REVIEW

**Comment by PerrineAguiar** [comment]
@adamantivm @jubeira ptal

File: org/ros/rosjava_messages/tango_ros_messages/1.0.0/tango_ros_messages-1.0.0.pom~ [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.ros.rosjava_messages</groupId>
+  <artifactId>tango_ros_messages</artifactId>
+  <version>1.0.0</version>
+  <dependencies>
+    <dependency>
+      <groupId>org.ros.rosjava_bootstrap</groupId>
+      <artifactId>message_generation</artifactId>
+      <version>[0.2,0.3)</version>
```
Comment (adamantivm) [inline org/ros/rosjava_messages/tango_ros_messages/1.0.0/tango_ros_messages-1.0.0.pom~]: @PerrineAguiar will you correct this manually, in the end?

**Comment by adamantivm** [comment]
only one comment left @PerrineAguiar

File: org/ros/rosjava_messages/tango_ros_messages/1.0.0/tango_ros_messages-1.0.0.pom~ (hunk 1 above)
Comment (PerrineAguiar) [inline org/ros/rosjava_messages/tango_ros_messages/1.0.0/tango_ros_messages-1.0.0.pom~]: I already corrected it manually. This is a back file from gedit that I commited by mistake.

**Comment by PerrineAguiar** [comment]
@adamantivm Thanks for the review. I addressed your comment.

**Comment by adamantivm** [comment]
:clap:

**Comment by jubeira** [comment]
:tada:

**Comment by PerrineAguiar** [comment]
@adamantivm @jubeira Thanks a lot :)

**Pull request merged**